test/lib/ufe/CMakeLists.txt

```diff
@@ -54,6 +54,18 @@ if(CMAKE_UFE_V3_FEATURES_AVAILABLE)
     )
 endif()
 
+if (UFE_LIGHTS_SUPPORT)
+list(APPEND TEST_SCRIPT_FILES
+    testLight.py
+)
+endif()
```

Comment:
I'll remove this, my local changes didn't have the update at line 90 that does the same thing.